metrics-micrometer/src/main/java/org/axonframework/micrometer/PayloadTypeMessageMonitorWrapper.java

```diff
@@ -32,11 +32,14 @@
  * MessageMonitor}.
  *
  * @param <T> The type of the MessageMonitor created for every payload type.Must implement both {@link MessageMonitor}
- *
  * @author Steven van Beelen
  * @author Marijn van Zelst
  * @since 4.1
+ * @deprecated As of release 4.4, replaced by specific monitors {@link MessageCountingMonitor}, {@link CapacityMonitor},
+ * {@link EventProcessorLatencyMonitor}, {@link MessageCountingMonitor} that can be used to register meters on Axon
+ * components directly.
```

Comment:
I believe this is purely deprecated because of the use of `Tags` right?
So, I would expect something like this.

```suggestion
 * @deprecated As of release 4.4, replaced by using {@link Tag}s on the monitor implementations.
 * Use {@link org.axonframework.micrometer.GlobalMetricRegistry#registerWithConfigurerWithDefaultTags(Configurer) to achieve the same behavior as this implementation.
```

Reply:
Yes, correct. No need for the `PayloadTypeMessageMonitorWrapper.java` any more